### PR TITLE
add event query, alert and notification docs

### PR DESCRIPTION
**Pull request by junotx**
add event query, alert and notification docs

/assign @FeynmanZhou

### CODE REVIEW

File: content/en/docs/project-user-guide/alerting/alerting-policy.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+### Task 3: Select Monitoring Targets
+
+Supports three workloads as monitoring targets: deployment, statefulset and daemonset. 
+Here select **Deployments** and select `reviews-v1` and `details-v1` as monitoring targets. Then click **Next**.
+
+![alerting_policy_workload_level_monitoring_target](/images/docs/alerting/alerting_policy_workload_level_monitoring_target.png)
+
+### Task 4: Add Alerting Rules
+
+Click **Add Rule**. As an example, you can create a rule with `memory usage`, `1 minute/period`, `2 consecutive times` and memory threshold `>` `20` MiB and set the rule level to `Major Alert`. Click **√** to save when finish that, and then click **Next**.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/alerting/alerting-policy.md:47]: Save as above.

->

Click **Add Rule** to begin to create an alerting rule. The rule defines parameters such as metric type, check period, consecutive times, metric threshold and alert level to provide rich configurations. The check period (the second field under **Rule**) means the time interval between 2 consecutive checks of the metric. For example, 2 minutes/period means the metric is checked every two minutes. The consecutive times (the third field under **Rule**) means the number of consecutive times that the metric meets the threshold when checked. An alert is only triggered when the actual time is equal to or is greater than the number of consecutive times set in the alert policy.

In this example, set those parameters to `memory usage`, `1 minute/period`, `2 consecutive times`, `>` and `20 MiB` for threshold and `Major Alert` for alert level. It means KubeSphere checks the memory usage every minute, and a major alert is triggered if it is larger than 20 MiB for 2 consecutive times.

Click √ to save the rule when you finish it, and click **Next**.

File: content/en/docs/project-user-guide/alerting/alerting-policy.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+![alerting_policy_workload_level_alerting_rule](/images/docs/alerting/alerting_policy_workload_level_alerting_rule.png)
+
+{{< notice note >}}
+
+- The metrics supported by workload-level alert policy are classified as follows:
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/alerting/alerting-policy.md:53]: ->
You can create workload-level alert policies for the following metrics:

File: content/en/docs/project-user-guide/alerting/alerting-message.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Hands-on Lab
+
+### Task 1: View Alert Message
+
+1. Log in the console and go to your project. Navigate to **Alerting Message** under **Monitoring & Alerting**, and you can see alert messages in the list. In the example of [Alert Policy - workload level](../alerting-policy/) you set two monitoring targets `reviews-v1` and `details-v1`, and both of their memory usages are higher than the threshold, so you can see two alert messages corresponding to them.
+
+![alerting_message_workload_level_list](/images/docs/alerting/alerting_message_workload_level_list.png)
+
+2. Select one of the alert messages to enter the details page, and in **Alert Detail** you can see the graph of the memory usage of the monitored workload over time, which has been continuously higher than the threshold of 20 MiB set by the alert rule, so the alert was triggered.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/alerting/alerting-message.md:24]: ->
Select one of the alert messages to enter the detail page. In **Alerting Detail**, you can see the graph of the memory usage of the monitored workload over time, which has been continuously higher than the threshold of 20 MiB set in the alert rule, so the alert was triggered.

File: content/en/docs/project-user-guide/alerting/alerting-message.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+![alerting_message_workload_level_detail](/images/docs/alerting/alerting_message_workload_level_detail.png)
+
+### Task 2: View Alert Policy
+
+Switch to **Alert Policy** to view the alert policy corresponding to this alert message, and you can see triggering rule of it set in the example of [Alert Policy - workload level](../alerting-policy/).
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/alerting/alerting-message.md:30]: ->
Switch to **Alerting Policy** to view the alert policy corresponding to this alert message, and you can see the triggering rule of it set in the example of [Alert Policy - workload level](../alerting-policy/).

**Review by Sherlock113 — COMMENTED** [review]
Please refer to my comments and make changes. Thanks.

The process and logic look clear to me and I will update the entire wording after this pr is merged. Thanks.

**Comment by junotx** [comment]
The changes have been made and the conflicts solved @Sherlock113

**Comment by Sherlock113** [comment]
/lgtm
Please merge this pr and I will fix some format and wording issues. Thanks. @FeynmanZhou

**Pull request merged**